src/test/java/org/folio/service/mapping/TranslationFunctionUnitTest.java

```diff
@@ -139,6 +148,28 @@ void SetIdentifier_shouldReturnEmptyString_whenMetadataIsEmpty() {
   }
 
 
+  @Test
+  void SetMaterialType_shouldReturnMaterialTypeValue() {
+    // given
+    TranslationFunction translationFunction = TranslationsHolder.lookup("set_material_type");
+    String value = "1a54b431-2e4f-452d-9cae-9cee66c9a892";
+    // when
+    String result = translationFunction.apply(value, 0, null, referenceData, null);
+    // then
+    Assert.assertEquals("book", result);
+  }
+
+  @Test
+  void SetMaterialType_shouldReturnEmptyString() {
```

Comment:
i think method name should start with lower case letter...but it is ok

Reply:
Let's make an exclusion for enumeration tests as far as other test cases have the same name convention, can discuss this point internally for the future to have the common convention